packages/styles/carbon.yml

```diff
@@ -0,0 +1,681 @@
+# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas@v1/carbon-resources.schema.json
+library:
+  id: ibmdotcom-styles
+  name: IBM.com Styles
+  description: IBM.com framework agnostic component styles
+  noIndex: true
+  externalDocsUrl: www.ibm.com/standards/web/carbon-for-ibm-dotcom
+assets:
+  back-to-top:
+    name: Back to top
+    description: Back to top is a sticky button that appears on long form pages. When triggered it scrolls the the user back to the top of the page.
+    externalDocsUrl: https://www.ibm.com/standards/carbon/components/back-to-top
+    status: stable
+    type: component
+    platform: web
+    noIndex: true
+    thumbnailPath: ../../thumbnails/back-to-top.svg
```

Comment:
Makes sense to put all thumbnails in this one directory for starters. We might want to move them into `/packages/styles` asset subdirectories if each asset has a `/docs` directory in the future for usage, design, and code docs. But until we get there, I like this simple approach.